recipe/meta.yaml

```
@@ -13,9 +13,9 @@ source:
   sha256: 1db7f3af66e5dd8b63fb24bde737228c25c7b97a1d40b0597ff4ce1c4b3b0437
 
 build:
-  merge_build_host: true  # [win]
+  merge_build_host: true  # [win or ppc64le]
   skip: true  # [win]
```

I believe you wanted:

```suggestion
  merge_build_host: true  # [win]
  skip: true  # [win or ppc64le]
```

---

ooopsy :(  Thank you.